quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/Picocli.java

```diff
@@ -528,7 +539,26 @@ private static void handleDisabled(Set<String> disabledInUse, PropertyMapper<?>
         }
         disabledInUse.add(sb.toString());
     }
-    
+
+    private static void handleRequired(Set<String> requiredOptions, PropertyMapper<?> mapper) {
+        String optionName = mapper.getFrom();
+        if (optionName.startsWith(NS_KEYCLOAK_PREFIX)) {
+            optionName = optionName.substring(NS_KEYCLOAK_PREFIX.length());
+        }
+
+        final StringBuilder sb = new StringBuilder("\t- ");
+        sb.append(optionName);
+        mapper.getRequiredWhen().ifPresent(msg -> {
+            sb.append(": ");
+            sb.append(msg);
+            if (!msg.endsWith(".")) {
```

Comment:
It like like from the other aggregation of required / enabled that we explicitly don't expect the message to end with a '.'. We can remove this check and even assert that isEnabled is called with string that does not end in '.'.